pysat/_instrument.py

```diff
@@ -265,27 +255,11 @@ def __init__(self, platform=None, name=None, tag=None, inst_id=None,
         # assign null data for user selected data type
         self.data = self._null_data.copy()
 
-        # create Meta instance with appropriate labels
-        self.units_label = units_label
-        self.name_label = name_label
-        self.notes_label = notes_label
-        self.desc_label = desc_label
-        self.plot_label = plot_label
-        self.axis_label = axis_label
-        self.scale_label = scale_label
-        self.min_label = min_label
-        self.max_label = max_label
-        self.fill_label = fill_label
-        self.meta = pysat.Meta(units_label=self.units_label,
-                               name_label=self.name_label,
-                               notes_label=self.notes_label,
-                               desc_label=self.desc_label,
-                               plot_label=self.plot_label,
-                               axis_label=self.axis_label,
-                               scale_label=self.scale_label,
-                               min_label=self.min_label,
-                               max_label=self.max_label,
-                               fill_label=self.fill_label)
+        # Create Meta instance with appropriate labels.  Meta class methos will
+        # use Instrument definition of MetaLabels over the Metadata declaration
+        self.meta_labels = labels
+        self.meta = pysat.Meta(labels=self.meta_labels)
+        self.labels = pysat.MetaLabels(metadata=self.meta, **labels)
```

Comment:
Initial impression. Shouldn't we pass in MetaLabels to Meta to define the labels, rather than the dict?

Reply:
Nope.  It requires the dict.  Not sure if we really need the labels definition here, but I'm trying not to break everything :P